File: 10 - Python/Exercícios/01 - Estrutura Sequencial/Exercício 05.py
#Exercício - Troco

preco = float(input("Preço unitário do produto: "))
quantidade = int(input("Quantidade comprada: "))
dinheiro = float(input("Dinheiro recebido: "))

troco = dinheiro - preco * quantidade

print("Troco = R$", troco)

File: 06 - C/Exercícios/01 - Estrutura Sequencial/Exercício 05.c
//Exercício - Troco

#include <stdio.h>

int main() {

    int quantidade;
    double preco, dinheiro, troco;

    printf("Preco unitario do produto: ");
    scanf("%lf", &preco);

    printf("Quantidade comprada: ");
    scanf("%d", &quantidade);

    printf("Dinheiro recebido: ");
    scanf("%lf", &dinheiro);

    troco = dinheiro - (preco * quantidade);

    printf("Troco = R$%.2lf", troco);

    return 0;

}